src/libraries/Common/src/Interop/Unix/System.Globalization.Native/Interop.Normalization.cs

```diff
@@ -12,7 +13,19 @@ internal static partial class Globalization
         [DllImport(Libraries.GlobalizationNative, CharSet = CharSet.Unicode, EntryPoint = "GlobalizationNative_IsNormalized")]
         internal static extern int IsNormalized(NormalizationForm normalizationForm, string src, int srcLen);
 
+        internal static int NormalizeString(NormalizationForm normalizationForm, string src, int srcLen, Span<char> dstBuffer)
```

Comment:
This wrapper does not seem to be making things better. Given that this is only used in one place, I think it would be better to just inline it into the one place where it is used.

Reply:
Changed for all calls